library/public/log.h

```diff
@@ -101,8 +101,10 @@ class F3D_EXPORT log
 
   /**
    * Set the verbose level.
+   * By default, only warnings and errors are written to stderr, debug and info are written to
+   * stdout. If forceStdErr is true, all messages including debug and info are written to stderr.
    */
-  static void setVerboseLevel(VerboseLevel level);
+  static void setVerboseLevel(VerboseLevel level, bool forceStdErr = false);
```

Comment:
Does adding another staticc method makes sense ?

Reply:
That was my initial plan but it is calling the same function so exposing the display stream mode makes more sense imho

Reply:
right I get your point.